<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <item>
      <widget class="QFrame" name="Navbar">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(253, 231, 189)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="backButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border : None;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/icon/button_back.png</normaloff>../img/icon/button_back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>70</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="homeButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>0</y>
          <width>231</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border : None;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/icon/logo.png</normaloff>../img/icon/logo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>190</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1201</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 229);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1176</width>
          <height>1129</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>1000</width>
             <height>600</height>
            </size>
           </property>
           <widget class="QLabel" name="fotoArtikel">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>30</y>
              <width>1051</width>
              <height>441</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 8px;
border-style: outset;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../img/ayam.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="namaArtikel">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>480</y>
              <width>901</width>
              <height>101</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 24pt;</string>
            </property>
            <property name="text">
             <string>Ayam Goreng adalah Makanan yang Enak Sekali</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>50</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>500</height>
            </size>
           </property>
           <widget class="QLabel" name="tanggal">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>23 February 2023</string>
            </property>
           </widget>
           <widget class="QLabel" name="textEdit">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>60</y>
              <width>1031</width>
              <height>421</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
